docker/complement/Dockerfile

```diff
@@ -8,16 +8,16 @@
 
 ARG SYNAPSE_VERSION=latest
 
-# first of all, we create a base image with a postgres server and database,
-# which we can copy into the target image. For repeated rebuilds, this is
-# much faster than apt installing postgres each time.
+# First of all, we copy postgres server from the official postgres image,
+# since for repeated rebuilds, this is much faster than apt installing
+# postgres each time.
 #
 # This trick only works because (a) the Synapse image happens to have all the
 # shared libraries that postgres wants, (b) we use a postgres image based on
 # the same debian version as Synapse's docker image (so the versions of the
 # shared libraries match).
 
-# now build the final image, based on the Synapse image.
+# We also initialize the database itself at build and not runtime for speed reason.
 
 FROM matrixdotorg/synapse-workers:$SYNAPSE_VERSION
     # copy the postgres installation over from the image we built above
```

Comment:
what image we built above?

Reply:
I am not sure what you meant, but the deleted line is unrelated to the added one.
The deleted one is not really useful this we don't build a first stage for postgres now.

I moved the comment nearby the code as suggested so it should be clearer now.

Reply:
> I am not sure what you meant

I was commenting on this line:

```
    # copy the postgres installation over from the image we built above
```

you've removed it now.